bodhi/tests/server/functional/test_stacks.py

```diff
@@ -13,7 +13,11 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 import mock
+import copy
 
+from webtest import TestApp
```

Comment:
Similar import groupings here - let's put ```copy``` by itself in the first group, and let's group ```mock``` and ```webtest``` together.